openlibrary/plugins/worksearch/subjects.py

```diff
@@ -162,6 +162,16 @@ def normalize_key(self, key):
     def process_key(self, key):
         return key
 
+class subject_history(delegate.page):
+    path = '/subjects/history'
+    encoding = 'json'
+    
+    @jsonapi
+    def GET(self,key):
+        i = web.input()
+        if i.get("limit")>100:
```

Comment:
It might be a good idea to split this magic number into someplace else.

Reply:
OK, done.